mono_build_scripts_linux.sh

```
@@ -0,0 +1,20 @@
+#/bin/bash
```

Shellcheck (0.11.0)

[error] Use #!, not just #, for the shebang.

(SC1113)